test/notifications/display_test.dart

```diff
@@ -121,6 +122,7 @@ void main() {
     addTearDown(testBinding.reset);
     testBinding.firebaseMessagingInitialToken = '012abc';
     addTearDown(NotificationService.debugReset);
+    addTearDown(NotificationNavigationService.debugReset);
```

Comment:
It'd be good if we can avoid going and adding another detail like this that tests have to know about.

How about having `NotificationService.debugReset` call the new service's `debugReset`? Seems appropriate for the main service to take responsibility for resetting the new one, given that the main service is also the one place responsible for starting up the new service.